api/src/config.ts

```diff
@@ -1,5 +1,4 @@
-import logger from "./lib/logger";
-import { randomString } from "./service/hash";
+import { envVarsSchema } from "envVarsSchema";
```

Comment:
Absolute import will fail on pipeline and probably on someone else's machine too